js/deer-render.js

```diff
@@ -104,18 +104,35 @@ RENDER.element = function (elem, obj) {
  * @param {Object} obj some json to be drawn as JSON
  * @param {Object} options additional properties to draw with the JSON
  */
+
+function myFunction(jsonString) {
```

Comment:
This is not used anywhere.  Please remove it if it is not to be implemented in this PR.